code/style.css

```diff
@@ -1,58 +1,185 @@
+  justify-content: center;
+}
+
+.header-div {
+  color: #F2F2F2;
+  font-size: 35px;
+  font-weight: bold;
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.header-text {
+  background-color: transparent;
+  color: #F2F2F2;
+  font-weight: bold;
+  font-size: 25px;
+  text-align: center;
+  padding: 10px;
+}
+
+.instruction-text {
+  color: #F2F2F2;
+  font-style: italic;
+  text-align: center;
+  font-size: 18px;
+}
+
+.cards p {
   margin: 1px;
-  font-weight: 300;
-  background-color: #fff;
+  font-weight: 500;
+  background-color: inherit;
+  color: #F2F2F2;
+  font-size: 18px;
 }
 
 .cards {
-  margin: 50px auto;
+  margin: 35px auto;
   width: 90%;
   height: 600px;
-  background: #ccc;
+  background: #A6A6A6;
   display: flex;
   justify-content: space-around;
   align-items: center;
 }
 
+/**KILLER DECK**/
 .killer-deck {
   width: 250px;
   height: 400px;
-  background: #000;
+  background: #262626;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  border: 1px solid #A6A6A6;
+}
+
+.killer-deck :first-of-type {
+  font-size: 28px;
+  font-weight: 800;
+  color: #F2F2F2;
+  background-color: #A6A6A6;
 }
 
+.killer-deck p {
+  color: #262626;
+  font-weight: 800; 
+}
+
+.killer-deck:hover {
+  box-shadow: 0 8px 16px 0 rgba(247, 243, 243, 0.411);
+  cursor: pointer;
+}
+
+
+/**WEAPON DECK**/
 .weapon-deck {
   width: 250px;
   height: 400px;
-  background: rgb(10, 53, 10);
+  background: #262626;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  border: 1px solid #A6A6A6;
+}
+
+.weapon-deck :first-of-type {
+  font-size: 28px;
+  font-weight: 800;
+  background-color: #A6A6A6;
+}
+
+.weapon-deck:hover {
+  box-shadow: 0 8px 16px 0 rgba(247, 243, 243, 0.411);
+  cursor: pointer;
 }
 
+
+/**ROOM DECK**/
 .room-deck {
   width: 250px;
   height: 400px;
-  background: midnightblue;
-
+  background: #262626;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  border: 1px solid #A6A6A6;
+}
+
+.room-deck :first-of-type {
+  font-size: 28px;
+  font-weight: 800;
+  background-color: #A6A6A6;
+}
+
+.room-deck:hover {
+  box-shadow: 0 8px 16px 0 rgba(247, 243, 243, 0.411);
+  cursor: pointer;
+}
+
+.mystery-reveal {
+  width: 90%;
+  margin: 50px auto;
 }
 
+h1 {
+  font-size: 25px;
+  color: white;
+  text-align: center;
+}
+
+/**BUTTON**/
 button {
   padding: 8px 16px;
-  background: crimson;
+  background: #0D0D0D;
+  color: #A6A6A6;
   border: none;
   font-size: 20px;
+  font-weight: 500;
+  background: #0D0D0D;
+  color: #A6A6A6;
   border-radius: 8px;
   display: block;
   margin: 0 auto;
+  margin-top: 60px;
+}
+
+button:hover { 
+  box-shadow: 0 8px 16px 0 rgba(247, 243, 243, 0.411);
+  cursor: pointer;
+  background: #A6A6A6;
+  color: #0D0D0D;
+}
+
+/**FOOTER**/
+footer {
+  background: black;
+  text-align: center;
+  margin-top: 60px;
+  padding: 25px;
+  position: relative;
+  bottom: 0;
+  right: 0;
+  left: 0;
```

Comment:
Do you remember when during first lectures I was telling all of you about proper order of this CSS properties?
The correct order (from semantical point of view of course) should be 1. top, 2. right, 3. bottom, 4. left
Basically, clockwise